client/blocks/comment-button/index.jsx

```diff
@@ -10,9 +10,16 @@ import './style.scss';
 
 const noop = () => {};
 
-function CommentButton( props ) {
-	const { commentCount, href, onClick, tagName, target, icon, defaultLabel } = props;
-
+function CommentButton( {
+	commentCount = 0,
+	href = null,
+	onClick = noop,
```

Comment:
I wonder if the `noop` is necessary. Seems like the `omitBy()` below will remove the `onClick` if it defaults to `null`. An `undefined` should also be OK, perhaps.

Reply:
Good point, I managed to remove all the `noop`s in the components in this PR 🙂 In this `CommentButton` I also removed the `omitBy` logic and started passing `undefined` as values of the props that have no value. This is fine, although TypeScript will start complaining one day, because `<button href={ undefined }>` is React-safe, but React TS types don't allow this.